src/main/java/fi/aalto/cs/apluscourses/intellij/utils/ListDependenciesPolicy.java

```diff
@@ -7,9 +7,9 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
- * This class is a {@RootPolicy} that builds a list of the names of those {@OrderEntry} objects that
- * represents dependencies of an {@IntelliJComponent} object (that is, modules and non-module-level
- * libraries).
+ * This class is a {@link RootPolicy} that builds a list of the names of those {@code OrderEntry}
+ * objects that represents dependencies of an {@code IntelliJComponent} object (that is, modules
+ * and non-module-level libraries).
```

Comment:
I guess all of the three here could be links since `RootPolicy` is linked as well

Reply:
I didn't realize before that you had to import them for the links to work

Reply:
I'm no JavaDoc expert, but I guess if the imports aren't actually used in code, then it's preferred to use the fully qualified name in the links, i.e. `{@link com.intellij.openapi.roots.OrderEntry}`